core/src/main/java/org/infinispan/commands/functional/ReadOnlyKeyCommand.java

```diff
@@ -18,6 +18,7 @@
 
 public final class ReadOnlyKeyCommand<K, V, R> extends AbstractDataCommand implements LocalCommand {
```

Comment:
So, read-only functions now shipped around the cluster instead bringing data locally and doing the read locally? Nice :)